django/core/jinja2/base.jinja

```diff
@@ -288,6 +288,21 @@
     <link rel="mask-icon" href="{{ static('favicon/safari-pinned-tab.svg') }}" color="#5bbad5">
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="theme-color" content="#ffffff">
+    {% block ogp_tags %}
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="https://www.comses.net/">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="The Network for Computational Modeling in the Social and Ecological Sciences (CoMSES Net)">
+    <meta property="og:description" content="CoMSES Net is an international open research community dedicated to fostering good practices for computational / agent based modeling.">
+    <meta property="og:image" content="/static/images/logo.png">
```

Comment:
refer to static assets in a jinja template using the incantation I shared earlier,

`{{ static("images/logo-comses.png") }}`

so the full line would be something like

```
<meta property="og:image" content="{{ static('images/logo-comses.png') }}">
```

For more info on why we dynamically generate these static links, 

https://docs.djangoproject.com/en/4.2/howto/static-files/#configuring-static-files

(though the syntax looks a little different since we are using Jinja templates instead of Django templates)

Reply:
Apologies, forgot to fix these. I just updated them and the pull request is updated.